cmd/ocm-backplane/login/login.go

```diff
@@ -405,3 +414,17 @@ func listNamespaces(clusterID, clusterName string, isHostedControlPlane bool) ([
 
 	return nsList, nil
 }
+
+// getBackplaneCNAME returns the DNS/CNAME resolution of the ocm backplane URL
+func getBackplaneCNAME(backplaneURL string) (string, error) {
+	backplaneDomain, err := url.Parse(backplaneURL)
+	if err != nil {
+		return "", fmt.Errorf("unable to extract the fqdn from the %s", backplaneURL)
+	}
+	fqdn := backplaneDomain.Hostname()
+	resolution, err := net.LookupCNAME(fqdn)
+	if err != nil {
+		return "", fmt.Errorf("unable to resolve the %s", fqdn)
```

Comment:
Has this been tested? 
If so, can you please paste the relevant output logs to the PR?

Reply:
@supreeth7 : I have updated the PR's description with the relevant output logs .